class SatoriError(Exception):
    pass


class AuthError(SatoriError):
    pass
